### PR TITLE
[Workers] Sunder Workers starter template

### PR DESCRIPTION
**Pull request by deadlypants1973**
closes #443

### CODE REVIEW

**Review by kristianfreeman — APPROVED** [review]

File: products/workers/src/content/get-started/quickstarts.md [hunk 1]
```diff
@@ -56,6 +56,12 @@ $ wrangler generate <new-project-name> <github-repo-url>
   repo="cloudflare/worker-template-router"
 />
 
+<WorkerStarter
+  title="Sunder Starter (Typescript)"
+  description="A batteries-included starter template using Sunder, Typescript, ESBuild, Jest and Sass. Uses Worker Sites for static assets."
```
Comment (lukeed) [inline products/workers/src/content/get-started/quickstarts.md]:
```suggestion
  description="A batteries-included starter template using Sunder, TypeScript, esbuild, Jest and Sass. Uses Worker Sites for static assets."
```

**Review by lukeed — CHANGES_REQUESTED** [review]

**Review by lukeed — APPROVED** [review]

File: products/workers/src/content/get-started/quickstarts.md [hunk 2]
```diff
@@ -56,6 +56,12 @@ $ wrangler generate <new-project-name> <github-repo-url>
   repo="cloudflare/worker-template-router"
 />
 
+<WorkerStarter
+  title="Sunder Starter (Typescript)"
+  description="A batteries-included starter template using Sunder, TypeScript, esbuild, Jest and Sass. Uses Worker Sites for static assets."
```
Comment (kodster28) [inline products/workers/src/content/get-started/quickstarts.md]:
```suggestion
  description="A batteries-included starter template using Sunder, TypeScript, esbuild, Jest, and Sass. Uses Worker Sites for static assets."
```

Comment (kodster28) [reply products/workers/src/content/get-started/quickstarts.md]:
```suggestion
  description="A complete starter template using Sunder, TypeScript, esbuild, Jest, and Sass. Uses Worker Sites for static assets."
```

Comment (kodster28) [reply products/workers/src/content/get-started/quickstarts.md]: Not sure if mine is the best term, but I don't think `batteries-included` won't generalize well to international audiences.

**Review by kodster28 — APPROVED** [review]

**Pull request merged**